Source/Core/DolphinQt/Settings/InterfacePane.cpp

```diff
@@ -144,11 +148,15 @@ void InterfacePane::CreateUI()
   m_checkbox_top_window = new QCheckBox(tr("Keep Window on Top"));
   m_checkbox_use_builtin_title_database = new QCheckBox(tr("Use Built-In Database of Game Names"));
   m_checkbox_use_userstyle = new QCheckBox(tr("Use Custom User Style"));
+  m_checkbox_use_covers = new QCheckBox(tr("Download Game Covers for Use in Grid Mode"));
   m_checkbox_show_debugging_ui = new QCheckBox(tr("Show Debugging UI"));
 
+  m_checkbox_use_builtin_title_database->setToolTip(tr("Covers provided by gametdb.com"));
```

Comment:
This should be more prominent. It's not obvious that this specific checkbox would have a tooltip.